src/app/update-card-modal/update-card-modal.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { FormBuilder, FormGroup, FormsModule, ReactiveFormsModule, Validators } from '@angular/forms';
+import { MatButtonModule } from '@angular/material/button';
+import { MatDialogModule, MatDialogRef } from '@angular/material/dialog';
+import { MatInputModule } from '@angular/material/input';
+import { MatProgressSpinnerModule } from '@angular/material/progress-spinner';
+import { RecaptchaComponent, RecaptchaModule } from 'ng-recaptcha';
+
+import { allChildComponentImports } from '../../allChildComponentImports';
+import {Card, PaymentMethod} from "@stripe/stripe-js";
+
+
+@Component({
+  standalone: true,
+  selector: 'app-update-card-modal',
+  templateUrl: 'update-card-modal.html',
+  styleUrls: ['./update-card-modal.component.scss'],
+  imports: [
+    ...allChildComponentImports,
+    FormsModule,
+    MatButtonModule,
+    MatDialogModule,
+    MatInputModule,
+    MatProgressSpinnerModule,
+    ReactiveFormsModule,
+    RecaptchaModule,
+  ],
+})
+export class UpdateCardModalComponent implements OnInit {
+  @ViewChild('captcha') captcha: RecaptchaComponent;
```

Comment:
I think we can get rid of the captcha - we're already logged in and have a payment card saved.